pkg/runtime/pubsub/adapter.go

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package pubsub
+
+import (
+	contrib_pubsub "github.com/dapr/components-contrib/pubsub"
+)
+
+// PubSubAdapter is the interface for message buses
+type PubSubAdapter interface {
+	PubSubFeatures(pubsubName string) ([]contrib_pubsub.Feature, error)
```

Comment:
This function has `PubSub` prefix since the implementation so implementation of this interface can also implement the `Features()` method for other building block adapters without conflicting in name.